src/__experimental__/components/label/label.style.js

```diff
@@ -8,6 +8,17 @@ const LabelStyle = styled.label`
   display: block;
   font-weight: 600;
 
+  ${({
+    isRequired
+  }) => isRequired && css`
+    ::after {
```

Comment:
Please can we confirm that this approach is acceptable for users using assistive technology, e.g. screen reader.

Reply:
https://www.accessibility-developer-guide.com/examples/forms/required/
https://www.deque.com/blog/anatomy-of-accessible-forms-required-form-fields/

Reply:
Yes I try with ChromeVox Classic Extension (free, based on aria-required props https://chrome.google.com/webstore/detail/chromevox-classic-extensi/kgejglhpjiefppelpmljglcjbhoiplfn)

Reply:
Done